sqlScripts/test.30.5.sql

```diff
@@ -0,0 +1,47 @@
+USE kodilla_course;
+
+CREATE TABLE stats 
+(
+	stat_id 	INT(11) AUTO_INCREMENT PRIMARY KEY,
+	stat_date 	DATETIME NOT NULL,
+	state 		VARCHAR(20) NOT NULL,
+	value INT(11) NOT NULL
+);
+
+SELECT * FROM stats;
+
+CREATE VIEW bestsellers_count AS
+SELECT COUNT(*) AS bestsellers
+FROM books
+WHERE bestseller = 1;
+
+DROP VIEW bestsellers_count;
+
+COMMIT;
+
+INSERT INTO STATS (state_date, stat, value)
+VALUES (CURRENT_TIMESTAMP, "Bestsellers", (SELECT * FROM bestsellers_count));
```

Comment:
to dziala? nie powinno by SELECT COUNT(*) ?

Reply:
jeśli dam SELECT COUNT(*)  będzie mi zwracać wartość 1 a powinien 3 (tyle jest bestsellerów w tabeli)